packages/amplify_datastore/ios/Classes/utils/modelHelpers.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Foundation
+import Amplify
+
+public func convertToAmplifyPersistable(value: Any?) -> Persistable? {
+    if value == nil {
+        return nil
+    }
+    switch value {
+    case is Int:
+        return value as! Int
+    case is Double:
+        return value as! Double
+    case is Bool:
+        return value as! Bool
+    case is String:
+        return value as! String
```

Comment:
I'd not force-unwrap (since it will crash the app if for any reason the casting fails) and instead return an optional value
i.e. `return value as? Int`

Or alternatively you can store the result of each type casting `as?` in a variable and after the switch throw an error if the casting failed